mteb/_evaluators/text/summarization_evaluator.py

```diff
@@ -201,12 +203,12 @@ def _calculate_metrics(
         self,
         distances: SummarizationDistances,
     ) -> SummarizationMetrics:
-        cosine_spearman_scores = []
-        cosine_pearson_scores = []
-        dot_spearman_scores = []
-        dot_pearson_scores = []
-        pearson_scores = []
-        spearman_scores = []
+        cosine_spearman_scores: list[float] = []
```

Comment:
is it important that these are list float (they could be list of anything that is valid for np.mean